tools/affine-cli/skills/affine/REFERENCE.md

```
@@ -0,0 +1,155 @@
+    [--fill "#rrggbb"] [--stroke "#rrggbb"] [--text "label inside shape"]
+
+affine-cli diagram add-text --workspace=<ws> --doc=<id> --text "<t>" [--xywh "[x,y,w,h]"] [--color "#rrggbb"]
+
+affine-cli diagram add-connector --workspace=<ws> --doc=<id> --from=<elementId> --to=<elementId> \
+    [--mode straight|elbow|curve]   # default elbow; invalid values error
+    [--label "<t>"]
+```
+
+High-level — build an entire graph from one JSON file:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)